{
  "uid" : "a991678ed86ca3a",
  "name" : "turn on light",
  "historyId" : "bbb22bd1e1e837820a07aa68780e17a1",
  "time" : {
    "start" : 1638915067984,
    "stop" : 1638915088589,
    "duration" : 20605
  },
  "status" : "failed",
  "statusMessage" : "AssertionError: Mycroft responded with:\nMycroft: unknown.dialog(UnknownSkill)\n\"Sorry, I didn't catch that.\"\nMycroft: pairing.code.dialog(PairingSkill)\n\"Enter the code. 'F' as in Fox trot. 'T' as in Tango. 'M' as in Mike. 'M' as in Mike. 'A' as in Apple. 'Y' as in Yankee.\"\n\n",
  "statusTrace" : "  File \"/opt/mycroft-core/.venv/lib/python3.7/site-packages/behave/model.py\", line 1806, in run\n    match.run(runner.context)\n  File \"/opt/mycroft-core/.venv/lib/python3.7/site-packages/behave/matchers.py\", line 97, in run\n    self.func(context, *args, **kwargs)\n  File \"features/steps/utterance_responses.py\", line 157, in then_dialog\n    assert passed, assert_msg or 'Mycroft didn\\'t respond'\n",
  "flaky" : false,
  "newFailed" : false,
  "beforeStages" : [ ],
  "testStage" : {
    "status" : "failed",
    "statusMessage" : "AssertionError: Mycroft responded with:\nMycroft: unknown.dialog(UnknownSkill)\n\"Sorry, I didn't catch that.\"\nMycroft: pairing.code.dialog(PairingSkill)\n\"Enter the code. 'F' as in Fox trot. 'T' as in Tango. 'M' as in Mike. 'M' as in Mike. 'A' as in Apple. 'Y' as in Yankee.\"\n\n",
    "statusTrace" : "  File \"/opt/mycroft-core/.venv/lib/python3.7/site-packages/behave/model.py\", line 1806, in run\n    match.run(runner.context)\n  File \"/opt/mycroft-core/.venv/lib/python3.7/site-packages/behave/matchers.py\", line 97, in run\n    self.func(context, *args, **kwargs)\n  File \"features/steps/utterance_responses.py\", line 157, in then_dialog\n    assert passed, assert_msg or 'Mycroft didn\\'t respond'\n",
    "steps" : [ {
      "name" : "Given an English speaking user",
      "time" : {
        "start" : 1638915067985,
        "stop" : 1638915067985,
        "duration" : 0
      },
      "status" : "passed",
      "steps" : [ ],
      "attachments" : [ ],
      "parameters" : [ ],
      "attachmentsCount" : 0,
      "shouldDisplayMessage" : false,
      "hasContent" : false,
      "stepsCount" : 0
    }, {
      "name" : "When the user says \"can you turn on Mycroft light please\"",
      "time" : {
        "start" : 1638915067985,
        "stop" : 1638915067985,
        "duration" : 0
      },
      "status" : "passed",
      "steps" : [ ],
      "attachments" : [ ],
      "parameters" : [ ],
      "attachmentsCount" : 0,
      "shouldDisplayMessage" : false,
      "hasContent" : false,
      "stepsCount" : 0
    }, {
      "name" : "Then \"homeassistant\" should reply with dialog from \"homeassistant.device.on.dialog\"",
      "time" : {
        "start" : 1638915067985,
        "stop" : 1638915077986,
        "duration" : 10001
      },
      "status" : "failed",
      "statusMessage" : "AssertionError: Mycroft responded with:\nMycroft: unknown.dialog(UnknownSkill)\n\"I'm sorry, I don't understand.\"\n\n",
      "statusTrace" : "  File \"/opt/mycroft-core/.venv/lib/python3.7/site-packages/behave/model.py\", line 1806, in run\n    match.run(runner.context)\n  File \"/opt/mycroft-core/.venv/lib/python3.7/site-packages/behave/matchers.py\", line 97, in run\n    self.func(context, *args, **kwargs)\n  File \"features/steps/utterance_responses.py\", line 157, in then_dialog\n    assert passed, assert_msg or 'Mycroft didn\\'t respond'\n",
      "steps" : [ ],
      "attachments" : [ ],
      "parameters" : [ ],
      "attachmentsCount" : 0,
      "shouldDisplayMessage" : true,
      "hasContent" : true,
      "stepsCount" : 0
    }, {
      "name" : "Given an English speaking user",
      "time" : {
        "start" : 1638915078287,
        "stop" : 1638915078287,
        "duration" : 0
      },
      "status" : "passed",
      "steps" : [ ],
      "attachments" : [ ],
      "parameters" : [ ],
      "attachmentsCount" : 0,
      "shouldDisplayMessage" : false,
      "hasContent" : false,
      "stepsCount" : 0
    }, {
      "name" : "When the user says \"can you turn on Mycroft light please\"",
      "time" : {
        "start" : 1638915078287,
        "stop" : 1638915078287,
        "duration" : 0
      },
      "status" : "passed",
      "steps" : [ ],
      "attachments" : [ ],
      "parameters" : [ ],
      "attachmentsCount" : 0,
      "shouldDisplayMessage" : false,
      "hasContent" : false,
      "stepsCount" : 0
    }, {
      "name" : "Then \"homeassistant\" should reply with dialog from \"homeassistant.device.on.dialog\"",
      "time" : {
        "start" : 1638915078287,
        "stop" : 1638915088288,
        "duration" : 10001
      },
      "status" : "failed",
      "statusMessage" : "AssertionError: Mycroft responded with:\nMycroft: unknown.dialog(UnknownSkill)\n\"Sorry, I didn't catch that.\"\nMycroft: pairing.code.dialog(PairingSkill)\n\"Enter the code. 'F' as in Fox trot. 'T' as in Tango. 'M' as in Mike. 'M' as in Mike. 'A' as in Apple. 'Y' as in Yankee.\"\n\n",
      "statusTrace" : "  File \"/opt/mycroft-core/.venv/lib/python3.7/site-packages/behave/model.py\", line 1806, in run\n    match.run(runner.context)\n  File \"/opt/mycroft-core/.venv/lib/python3.7/site-packages/behave/matchers.py\", line 97, in run\n    self.func(context, *args, **kwargs)\n  File \"features/steps/utterance_responses.py\", line 157, in then_dialog\n    assert passed, assert_msg or 'Mycroft didn\\'t respond'\n",
      "steps" : [ ],
      "attachments" : [ ],
      "parameters" : [ ],
      "attachmentsCount" : 0,
      "shouldDisplayMessage" : true,
      "hasContent" : true,
      "stepsCount" : 0
    } ],
    "attachments" : [ ],
    "parameters" : [ ],
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : true,
    "stepsCount" : 6
  },
  "afterStages" : [ ],
  "labels" : [ {
    "name" : "severity",
    "value" : "normal"
  }, {
    "name" : "suite",
    "value" : "behave"
  }, {
    "name" : "feature",
    "value" : "turn on"
  }, {
    "name" : "framework",
    "value" : "behave"
  }, {
    "name" : "language",
    "value" : "cpython3"
  }, {
    "name" : "resultFormat",
    "value" : "allure2"
  } ],
  "parameters" : [ ],
  "links" : [ ],
  "hidden" : false,
  "retry" : false,
  "extra" : {
    "severity" : "normal",
    "retries" : [ ],
    "categories" : [ {
      "name" : "Product defects",
      "matchedStatuses" : [ ],
      "flaky" : false
    } ],
    "tags" : [ ]
  },
  "source" : "a991678ed86ca3a.json",
  "parameterValues" : [ ]
}